pkg/jobs/jobs.go

```diff
@@ -235,6 +238,11 @@ func (j *Job) FractionProgressed(ctx context.Context, progressedFn FractionProgr
 					fractionCompleted, j.id,
 				)
 			}
+			if p, ok := progress.Progress.(*jobspb.Progress_FractionCompleted); ok &&
+				p.FractionCompleted == fractionCompleted {
```

Comment:
Are we okay with float equality here? I'm guessing yes, but just wanted to check.

Reply:
Yeah, I'm mainly targetting the case where nothing has changed in whatever formula is used.